### ISSUE TITLE
netVLAD keras 구현 코드

### ISSUE THREAD

**Issue by glee1228** [suimn416/2019.Spring.AI_Leader#20]
https://colab.research.google.com/drive/1OyTXv6IG5E1Uq1ASOLUc9dz-kyDvu-Si

[케라스 netVLAD+TripletLoss 코드]
웹 상에 아직 전체적으로 구현되지 않은 netVLAD+triplet loss 코드를 구현하고 있습니다. 

데이터를 generator로 input 하는 과정 중에 있습니다. 
코드는 현재 80퍼센트 정도 완성하였고, 코드는 pytorch-netVLAD와 keras로 구현된 tripletNet을 GIthub에서 참고하여 작성하였습니다.

링크는 추후에 업로드 하겠습니다.

작성 예정 : pretrained 모델을 임포트해 사용할 수있도록 완성, 파이토치 SVM 분류를 다른 방식으로 작성할 필요성이 있음. 

빠른 작성이 목적이었으나 구현 시간이 지체되었습니다.최대한 빨리 작성하겠습니다.

**Comment by glee1228** [comment]
https://drive.google.com/file/d/1OyTXv6IG5E1Uq1ASOLUc9dz-kyDvu-Si/view?usp=sharing 
전체 보기 가능한 링크로 공유하겠습니다.

**Comment by Junuu** [comment]
넵 확인해보겠습니다.
